components/organisms/o-review-modal.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="o-review-modal">
+    <SfModal :visible="isModalVisible" @close="closeModal">
+      <div class="form">
+        <SfInput
+          v-model="formData.name"
+          name="firstName"
+          :label="$t('First name')"
+          :required="true"
+          :valid="!$v.formData.name.$error"
+          :error-message="
+            !$v.formData.name.required
+              ? $t('Field is required.')
+              : $t('Name must have at least 2 letters.')
+          "
+          class="form__input"
+        />
+        <SfInput
+          v-model="formData.email"
+          name="email"
+          :label="$t('Email address')"
+          :required="true"
+          :valid="!$v.formData.email.$error"
+          :error-message="
+            !$v.formData.email.required
+              ? $t('Field is required.')
+              : $t('Please provide valid e-mail address.')
+          "
+          class="form__input"
+        />
+        <SfInput
+          v-model="formData.summary"
+          name="summary"
+          :label="$t('Summary')"
+          :required="true"
+          :valid="!$v.formData.summary.$error"
+          :error-message="
+            !$v.formData.summary.required
+              ? $t('Field is required.')
+              : ''
+          "
+          class="form__input"
+        />
+        <SfInput
+          v-model="formData.review"
+          name="reviewText"
+          :label="$t('Review')"
+          :required="true"
+          :valid="!$v.formData.review.$error"
+          :error-message="
```

Comment:
podobnie jak wyżej, tutaj error message może być po prostu takie `:error-message="$t('Field is required.')"`

Reply:
done